#Message where each sub command from joy or keyboard message for the MQS is published into

uint8[16] cmd_ctrls
